google/api_core/extended_operation.py

```diff
@@ -137,9 +145,11 @@ def cancelled(self):
         self._refresh_and_update()
         return self._extended_operation.done
 
-    def _refresh_and_update(self, retry=polling.DEFAULT_RETRY):
+    def _refresh_and_update(self, retry=None):
         if not self._extended_operation.done:
-            self._extended_operation = self._refresh(retry=retry)
+            self._extended_operation = (
+                self._refresh(retry=retry) if retry else self._refresh()
```

Comment:
Simplify with self._refresh(retry=retry)

Reply:
if `retry=None` then `self._refresh(retry=retry)` and `self._refresh()` are different things, becuase `_refresh` is  most likely defined as `_refrhesh(retry=DEFAULT)`. Note in the whole semantics of gapics `retry/polling` configs `None` has special meaning, and is different from being "not set" or "default".